### PR TITLE
Added Bubble Sort Algorithm  in shell

### PR DESCRIPTION
**Pull request by karthikeyanrathore**
Files Added in : Sorting/Bubble Sort/Shell/bubble_sort.sh
Implemented Bubble Sort in Shell.

### CODE REVIEW

File: Sorting/Bubble Sort/Shell/bubble_sort.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Creating an array
+a=(200 100 300 700 600 500)
+n=6
+
+echo "Array Before Sorting: "
+echo ${a[*]}
+
+for ((i = 0; i < n; ++i))
+do
+	for ((j = 0; j < n - i - 1; ++j))
```
Comment (thevirtualbuddy) [inline Sorting/Bubble Sort/Shell/bubble_sort.sh:12]: Add a boolean variable before the inner for loop (say): 
`check = false`

File: Sorting/Bubble Sort/Shell/bubble_sort.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# Creating an array
+a=(200 100 300 700 600 500)
+n=6
+
+echo "Array Before Sorting: "
+echo ${a[*]}
+
+for ((i = 0; i < n; ++i))
+do
```
Comment (thevirtualbuddy) [inline Sorting/Bubble Sort/Shell/bubble_sort.sh:11]: Add a boolean variable before the inner for loop (say): 
`check = false`

File: Sorting/Bubble Sort/Shell/bubble_sort.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	for ((j = 0; j < n - i - 1; ++j))
+	do
+		if [ ${a[j]} -gt ${a[$((j + 1))]} ]
+		then 
+			temp=${a[j]}
+			a[$j]=${a[$((j + 1))]}
+			a[$((j + 1))]=$temp
```
Comment (thevirtualbuddy) [inline Sorting/Bubble Sort/Shell/bubble_sort.sh:18]: Make the boolean variable as true when swapping is done:
`check=true`

File: Sorting/Bubble Sort/Shell/bubble_sort.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+		fi
+	done
```
Comment (thevirtualbuddy) [inline Sorting/Bubble Sort/Shell/bubble_sort.sh:20]: Check if the boolean variable is true or false, if true break from the loop.

**Review by thevirtualbuddy — COMMENTED** [review]
All good. Added comments for optimization.

**Comment by karthikeyanrathore** [comment]
ok

File: Sorting/Bubble Sort/Shell/bubble_sort.sh (hunk 4 above)
Comment (karthikeyanrathore) [reply Sorting/Bubble Sort/Shell/bubble_sort.sh:20]: if `check = false` , them i should break from the loop right.

**Comment by karthikeyanrathore** [comment]
@thevirtualbuddy Please review my changes

File: Sorting/Bubble Sort/Shell/bubble_sort.sh (hunk 4 above)
Comment (thevirtualbuddy) [reply Sorting/Bubble Sort/Shell/bubble_sort.sh:29]: Yes, right! My bad. I check==false, break from the loop.

**Review by thevirtualbuddy — APPROVED** [review]
According to me, this is good to go.

**Comment by thevirtualbuddy** [comment]
@karthikeyanrathore Can you review PR #1285 ?

**Pull request merged**